src/averageAges.js

```diff
@@ -14,12 +19,20 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = people.filter(person => person.sex === 'm');
+
+  const menBasedCentury = century
+    ? men.filter(person => century === Math.ceil(person.died / 100))
+    : men;
+
+  return calcAvg(menBasedCentury);
```

Comment:
I know its passing tests but try to round the resulting average ages at all three returns! 
prepare `roundAge `function and use
here for example - `return roundAge(calcAvg(menBasedCentury));`  will give you answer in round number